pyogrio/tests/conftest.py

```diff
@@ -123,28 +124,165 @@ def naturalearth_lowres_vsi(tmp_path, naturalearth_lowres):
 
 
 @pytest.fixture(scope="session")
-def test_fgdb_vsi():
-    return f"/vsizip/{_data_dir}/test_fgdb.gdb.zip"
```

Comment:
Does it matter that we don't have a direct replacement for a zipped FGB?

Reply:
I don't think it matters.  We have other tests that use the `/vsizip/` interface for working with a zipped shapefile, which should be a reasonable proxy for zip files containing other formats.